main/acle.md

```diff
@@ -2699,11 +2704,26 @@ following:
   in one of the translation units.
   * Implicitly, without this attribute,
   * or explicitly providing the `default` in the attribute.
-* All instances of the versions shall share the same function
-  signature and calling convention.
+* The default version signature is the signature for calling
+  the multiversioned functions.
+* Non-default versions shall share the same calling convention
+  of the default version and have a type that is convertible to the type
+  of the default version.
+
 * All the function versions must be declared at the translation
   unit in which the definition of the default version resides.
 
+For example, the below is valid and 2 is passed as the
+value for `c` when calling `fn`.
+
+```C
+int __attribute__((target_version("simd"))) fn (int c = 1);
+int __attribute__((target_version("default"))) fn (int c = 2);
+int __attribute__((target_version("sve"))) fn (int c = 3);
+
+int baz() { return fn(); }
```

Comment:
Let's rename baz to something else. I suggest either use naming scheme `g` for caller and `f` for callee. Or just rename baz to `caller`. Otherwise LGTM

Reply:
Awesome, done! Thank you so much for your help with this.